src/actinia_core/core/common/process_chain.py

```diff
@@ -279,6 +278,57 @@ def _get_sentinel_import_command(self, entry):
 
         return sentinel_commands
 
+    def _get_sentinel_import_commands(self, entries):
```

Comment:
Maybe you could split this rather large method again? Or pull out the first part as `_get_setinel_scenen_info` or similar, so that it is not so big.

Reply:
This method is now split up